website/docs/user-group-role/roles/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: About roles
+---
+
+import DocCardList from "@theme/DocCardList";
+import { useCurrentSidebarCategory } from "@docusaurus/theme-common";
+
+Roles are a way to simplify the assignment of permissions. Roles are also the backbone of role-based access control (RBAC), an industry standard for managing [access control](../access-control). In authentik, RBAC is how you manage access to system components and specific objects such as flows, stages, and users.
+
+Think of roles as a collection of permissions. A role, along with its "bucket" of assigned permissions, can then be assigned to a group, which means that every user who is a part of that group will inherit all of the permissions in that role's "bucket".
+
+For example, let's take a look at the following scenario:
+
+> You need to add 5 new users, all new hires, to authentik, your identity management system. These users will be the first team members on the brand new Security team, so they will need some high-level permissions, with object permissions to create and remove other users, revoke permissions, and send recovery emails. They will also need [global permissions](../access-control/permissions#fundamentals-of-authentik-permissions) to control access to flows and stages.
```

Comment:
"... so they will need high-level access to other users in authentik, with permission to create and remove users, revoke permissions on those users, and send recovery emails."  

I almost changed the last "permissions" to "privileges," because my initial reaction was that the repetition of "permissions" was causing my eyes to glaze over, but it's better to keep it than introduce a new word for a concept we're trying to solidify in the user's head.

Reply:
it's a lot, I agree.